README.md

```diff
@@ -32,6 +32,3 @@ Docker image to initialize, backup to, and restore from a Restic repository on B
 
 It's recommended that your B2 bucket **NOT** have versioning turned on.
 Old versions of Restic's repository files are not useful.
-
```

Comment:
This is still being pushed there from what I can tell: https://github.com/sil-org/restic-backup-restore-b2/blob/30095da345c2fcea26087cfee08739058f84f29a/.github/workflows/build-and-publish.yml#L34